saleor/tests/API/checkout/test_unlogged_customer_should_be_able_to_order_digital_product.py

```diff
@@ -0,0 +1,109 @@
+from ..channel.utils import create_channel
+from ..product.utils import (
+    create_category,
+    create_digital_content,
+    create_digital_product_type,
+    create_product,
+    create_product_channel_listing,
+    create_product_variant,
+    create_product_variant_channel_listing,
+)
+from ..shipping_zone.utils import create_shipping_zone
+from ..warehouse.utils import create_warehouse
+from .utils import (
+    checkout_billing_address_update,
+    checkout_complete,
+    checkout_create,
+    checkout_dummy_payment_create,
+)
+
+
+def test_process_checkout_with_digital_product(
```

Comment:
Maybe each test should come with a comment or a docstring explaining the purpose or the test scenario? Could be useful for maintenance of such tests if we have more of them, and many will be similar but testing slightly different things? @saleor/qa What do you think?

Reply:
Or not to copy the scenario, paste a link to Testmo? So that there is some connection between the test and the scenario.

Reply:
Not everyone has access to testmo, so comments may be more useful here. I think it should at least have a test case ID (e.g for this test CORE_0101) and in case the title isn't descriptive enough we can provide more context in a comment.